### ISSUE TITLE
SwiftFormat 0.49.10 inserts unexpected indentation when using forward slash operator

### ISSUE THREAD

**Issue by konomae** [nicklockwood/SwiftFormat#1199]
An unexpected indent was inserted in a project using [CasePath](https://github.com/pointfreeco/swift-case-paths).

SwiftFormat 0.49.10
Config: default

This does not occur in 0.49.9 or earlier.

# Input

```swift
reducer.pullback(
    casePath: /Action.action,
    environment: {}
)
```

# Actual

```swift
reducer.pullback(
    casePath: /Action.action,
        environment: {}
    )
```

# Expected

same as input

```swift
reducer.pullback(
    casePath: /Action.action,
    environment: {}
)
```

**Comment by nicklockwood** [comment]
Thanks for reporting. Looks like the / is being mistaken for the start of a regex literal. Workaround for now would be to disable the indent rule for that file.

**Comment by nicklockwood** [comment]
@konomae fixed in 0.49.11

**Comment by konomae** [comment]
Thank you!
